ansible/group_vars/infra/dof_mongo.yml

```
@@ -1,5 +1,5 @@
 ---
 MONGO_IMAGE_NAME: mongo
-MONGO_IMAGE_TAG: 5.0.26
+MONGO_IMAGE_TAG: 6.0.15
```

Last version after docker hub is 6.0.16 
```suggestion
MONGO_IMAGE_TAG: 6.0.16
```